### PR TITLE
Reduce CTA arrow spacing, bring closer to text

### PR DESCRIPTION
**Pull request by starryeyez024**
Without theme: 
image

With RH theme:
image

[Design spec](https://xd.adobe.com/spec/3e62e93c-8338-4f31-5040-3b81f0ed5c71-4853/screen/f1027edf-aec8-476c-a763-576c508ba029/CTA-light-theme)

### Testing instructions

1. View the [CTA demo page](https://deploy-preview-844--happy-galileo-ea79c4.netlify.app//elements/pfe-cta/demo/) & compare the default CTA to the [mockups](https://xd.adobe.com/spec/3e62e93c-8338-4f31-5040-3b81f0ed5c71-4853/screen/f1027edf-aec8-476c-a763-576c508ba029/CTA-light-theme)
    - observe the spacing around the arrow, it should be closer to the text, as shown in the mockup
    - Test the hover and focus states
 
### Ready-for-merge Checklist

Check off items as they are completed.  Feel free to delete items if they are not applicable to your PR.

- [x] Expected files: all files in this pull request are related to one request or issue (no stragglers or scope-creep).
- [x] Browser testing passed.
- [x] Approved by designer.

### CODE REVIEW

**Comment by castastrophe** [comment]
@starryeyez024 Can you add a changelog when you get a chance?

**Comment by castastrophe** [comment]
### Testing instructions

Be sure to include detailed instructions on how your update can be tested by another developer.

1. 

#### Browser requirements

Your component should work in all of the following environments:

- [ ] Latest 2 versions of Edge
- [ ] Internet Explorer 11 (should be useable, not pixel perfect)
- [ ] Latest 2 versions of Firefox (one on Mac OS, one of Windows OS)
- [ ] Firefox 60.7.2 (or latest version for Red Hat Enterprise Linux distribution)
- [ ] Latest 2 versions of Chrome (one on Mac OS, one of Windows OS)
- [ ] Latest 2 versions of Safari
- [ ] Galaxy S9 Firefox
- [ ] iPhone X Safari
- [ ] iPad Pro Safari
- [ ] Pixel 3 Chrome

### Ready-for-merge Checklist

Check off items as they are completed.  Feel free to delete items if they are not applicable to your PR.

- [ ] Expected files: all files in this pull request are related to one request or issue (no stragglers or scope-creep).
- [ ] Tests have been updated to cover these changes.
- [ ] Browser testing passed.
- [ ] Documentation (README.md, WHY.md, etc.) updated or added.
- [ ] Link to the demo recording: []()
- [ ] Approved by designer.

File: CHANGELOG-prerelease.md [hunk 1]
```diff
@@ -11,6 +12,7 @@
 - [f9c2813](https://github.com/patternfly/patternfly-elements/commit/f9c2813892eace5e600d168d876ecc4ffd46b472) fix: Add a warning about updating the on attribute before upgrade (#809)
 - [a55449b](https://github.com/patternfly/patternfly-elements/commit/a55449bc1da11abdaf98f959961f25bfa20edc67) docs: Content set and tabs update (#824)
 - [eb74cb8](https://github.com/patternfly/patternfly-elements/commit/eb74cb8f989048164fbb6ed1508c502659a752ed) feat: Update pfe-select to use new event emission format (#758) (#760)
+- []() fix: adjust arrow spacing & alignment on pfe-cta
```
Comment (castastrophe) [inline CHANGELOG-prerelease.md:15]: I suspect this line is an accidental merge leftover?

Comment (starryeyez024) [reply CHANGELOG-prerelease.md]: ah yes thanks, fixed!

File: elements/pfe-cta/src/pfe-cta.scss [hunk 2]
```diff
@@ -7,10 +7,9 @@ $pfe-cta--BackgroundColor--focus: rgba(40, 151, 240, 0.2);
 $pfe-cta--Color--fallback: #003366;
 
 // Arrow animation
-$left: 0.2rem;
-$right: 0.2rem;
-$arrow-basic: 0 $right 0 $left;
-$arrow-hover: 0 0 0 #{$right + $left};
+$h-spacing: 3px;
```
Comment (castastrophe) [inline elements/pfe-cta/src/pfe-cta.scss:10]: @starryeyez024 What does h stand for?

Comment (starryeyez024) [reply elements/pfe-cta/src/pfe-cta.scss:10]: horizontal, i can spell it out if you prefer

File: elements/pfe-cta/src/pfe-cta.scss [hunk 3]
```diff
@@ -45,7 +51,7 @@ $variables: (
   position: relative;
   z-index: 0;
   vertical-align: middle;
-  transition: padding #{pfe-var(animation-speed)} #{pfe-var(animation-timing)};
+  //transition: padding #{pfe-var(animation-speed)} #{pfe-var(animation-timing)};
```
Comment (starryeyez024) [inline elements/pfe-cta/src/pfe-cta.scss:54]: @castastrophe  when i commented the other one out, it stopped animating. So i commented this one out (to test) and it seems to still work.. with no flicker. I'll send it out for another round of "can anyone see a flicker"

**Pull request merged**